deployed-contracts/core-pool.md

```diff
@@ -186,3 +186,17 @@
 * XVS Vault Proxy: [`0x407507DC2809D3aa31D54EcA3BEde5C5c4C8A17F`](https://sepolia.arbiscan.io/address/0x407507dc2809d3aa31d54eca3bede5c5c4c8a17f)
 * Prime: [`0xadb04ac4942683bc41e27d18234c8dc884786e89`](https://sepolia.arbiscan.io/address/0xadb04ac4942683bc41e27d18234c8dc884786e89)
 * Prime Liquidity Provider: [`0xe82c2c10f55d3268126c29ec813dc6f086904694`](https://sepolia.arbiscan.io/address/0xe82c2c10f55d3268126c29ec813dc6f086904694)
+
+## Zksync Sepolia
```

Comment:
Let's switch the order: first mainnet, then sepolia

Reply:
[a641fa1](https://github.com/VenusProtocol/venus-protocol-documentation/pull/224/commits/a641fa1f85c152de1f79e104492ec5fecbf16b04)